packages/ds-medicare-gov/src/styles/components/_SimpleFooter.scss

```diff
@@ -81,10 +74,12 @@
 }
 
 .m-c-footer__links {
-  margin-bottom: $spacer-3;
```

Comment:
being a good little boy scout and leaving this world a little better than i found it